### ISSUE TITLE
Call tracing result compatibility with Ethereum

### ISSUE THREAD

**Issue by jeongkyun-oh** [klaytn/klaytn#1618]
The result of call tracing is not compatible with Ethereum. In case of an external transaction (e.g. value transfer), the result of tracing is different as follows.

Klaytn (callTracer)

{noformat}{
"jsonrpc": "2.0",
"id": 1000,
"result": [
...,
{
"txHash": "0x906997bc1f35f72b474438cf0e94e037f562386d14aea45e82481809c9abbcca",
"result": {
"type": 0,
"from": "0x",
"to": "0x",
"value": "0x0",
"gas": "0x0",
"gasUsed": "0x0",
"input": "0x",
"output": "0x",
"time": 0
}
},
...,
]
}{noformat}

Ethereum (callTracer)

{noformat}{
"jsonrpc": "2.0",
"id": 1000,
"result": [
...,
{
"result": {
"type": "CALL",
"from": "0xc0ee6beee2c82d54fedae0e67bfb105b5139b045",
"to": "0x8627c9cc43f36ccb0ab5ec1ae05e845734af2c55",
"value": "0x235035b2a29080",
"gas": "0x0",
"gasUsed": "0x0",
"input": "0x",
"output": "0x"
}
},
...,
]
}{noformat}

ref: [https://forum.klaytn.foundation/t/klaytn-rpc-debug-traceblockbynumber/5944|https://forum.klaytn.foundation/t/klaytn-rpc-debug-traceblockbynumber/5944]

cc [~accountid:63d757a4614cb4ba53fdb39d] [~accountid:63d75167f1475ad42c5752f0]

**Comment by shilpachain** [comment]
We have the same issue while using **debug_traceTransaction** call. We used many nodes to execute this command, the result is same (Full node or Archival node)
```
curl -k https://klaytn03.fandom.finance/ \
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"debug_traceTransaction","params":["0xf00473214cd00d81c38d5ee9344d0c4e61ec4c278d12245b74fa8750aa5f3282", {"tracer":"callTracer"}],"id":1,"jsonrpc":"2.0"}' | jq

{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "type": 0,
    "from": "0x",
    "to": "0x",
    "value": "0x0",
    "gas": "0x0",
    "gasUsed": "0x0",
    "input": "0x",
    "output": "0x",
    "time": 0
  }
}
```
[Klaytn blockchain explorer](https://scope.klaytn.com/tx/0xf00473214cd00d81c38d5ee9344d0c4e61ec4c278d12245b74fa8750aa5f3282shows ) shows transfer of 80Klay but RPC response says it is 0x. In this response, from and to are also not populated. 


It seems the response is same for these types of transactions:
TX Type: **Fee Delegated Value Transfer**
Tx Type: **Legacy Transaction**

Will you please look into it or suggest alternatives? Thanks

**Comment by blukat29** [comment]
@shilpachain To query fields like `value`, `gas`, and `output` of the transactions that are not sent to contract accounts, you can use any of the below RPCs.

- [klay_getTransactionReceipt](https://docs.klaytn.foundation/dapp/json-rpc/api-references/klay/transaction#klay_gettransactionreceipt)
- [eth_getTransactionReceipt](https://docs.klaytn.foundation/dapp/json-rpc/api-references/eth/transaction#eth_gettransactionreceipt)

You can find the type of transaction ("type" field) from the outputs of the above RPCs.

**Comment by shilpachain** [comment]
Thank you @blukat29

**Comment by sjnam** [comment]
closed by #1953
